### PR TITLE
Update what-is-mesg.md

### PR DESCRIPTION
**Pull request by w1l1ams**
Combining Introduction, What is MESG?, Core, and MESG Infrastructure pages into a single page.

### CODE REVIEW

**Comment by w1l1ams** [comment]
@antho1404 @NicolasMahe - Please review. Thanks!

**Comment by NicolasMahe** [comment]
Do we need to keep the link like `[Core](./installation.md)`.
It doesn't make sense to have so many links to the install page.

Otherwise, good to merge 👍  @w1l1ams good job

**Comment by antho1404** [comment]
The links to installation are because before it was link to the page Core that has been deleted but yes maybe some of these links are simply not needed

File: docs/api/service.md [hunk 1]
```diff
@@ -38,12 +38,12 @@ The event and its data must be defined in the [service's definition file](../gui
...
-**Example:**
+Example:
 ```json
 {
-  "token":     "__SERVICE_TOKEN_FROM_ENV__",
-  "eventKey":  "__EVENT_KEY__",
-  "eventData": "{\"foo\":\"hello\",\"bar\":false}"
+"token":     "__SERVICE_TOKEN_FROM_ENV__",
+"eventKey":  "__EVENT_KEY__",
+"eventData": "{\"foo\":\"hello\",\"bar\":false}"
```
Comment (NicolasMahe) [inline docs/api/service.md]: @antho1404 I generate the doc for api and the template seems a bit different.
It remove the space at the beginning of the lines and the bold around example.
Do you know why?

Comment (antho1404) [reply docs/api/service.md]: try updating the branch ilker fixed that on the PR https://github.com/mesg-foundation/core/pull/541#discussion_r234115529

Comment (NicolasMahe) [reply docs/api/service.md]: thanks!

**Comment by NicolasMahe** [comment]
@antho1404 do we merge this branch directly to master or we wait for a new release (hopefully before the weekend)?

**Pull request merged**